### ISSUE TITLE
Increase the tap area for the scroll bar in Recent activity

### ISSUE THREAD

**Issue by Alex-302** [AdguardTeam/AdguardForAndroid#4765]
### Issue Details

When using a phone case, it is difficult to press the scroll bar slider with a finger and pull to scroll fast.

Related also to all screens where there is a list of settings.

### Proposed solution

Increase the width of the scroll bar by adding an invisible area to the left (+1 width of scroll bar)

### Alternative solution

In some applications, when you scroll with your finger through large lists, the size of the scroll slider increases and you can pull it.

**Comment by Nzzyme** [comment]
I agree. Scrolling on the Recent Activity page is currently very difficult by using the scroll slider. The slider width could be made thicker when the user taps on it.

**Comment by artemiv4nov** [comment]
We already improved a scroll slider at the right edge in previous versions, the task should be closed.

**Comment by maxikuzmin** [comment]
The problem still persists, it's really difficult to tap the scroll bar, even without a case on the device. We'll increase tap area for the scroll bar in Recent Activity in future updates

**Comment by Nzzyme** [comment]
Sadly, this is still not fixed. The scrolling feature (bar) in the app "Inure" is excellent. The same can be implemented in AG. Please check that app.

**Comment by Alex-302** [comment]
Checked with the last nightly - nothing changed. 
On screenshot - a typical swipe attempt, without any unnatural twisting of the finger or bending of the phone case's edge

<details><summary>Screenshot 1</summary>

<img width="576" height="1280" alt="Image" src="https://github.com/user-attachments/assets/9d0b4396-592f-4dba-9ab2-150426b4a353" />

</details>

**Comment by AntonVoronkin** [comment]
@Alex-302,
These changes have not reached the nightly version yet

**Comment by Alex-302** [comment]
@AntonVoronkin But many nights have already past.
To an outside observer, the essence of the solution of the problem remains unclear - was the slider width simply increased, or was it redesigned to resemble TrackChecker or Inure?

**Comment by AntonVoronkin** [comment]
The fix will be available today
